app/components/listing/feedback/FeedbackModal.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { Fragment, useState } from 'react';
+
+import {
+  FeedbackTags, Review, SubmitMessage, tagList,
+} from './FeedbackSteps';
+
+import { images } from '../../../assets';
+import './FeedbackModal.scss';
```

Comment:
Just as an update, in https://github.com/ShelterTechSF/askdarcel-web/pull/998 I changed how CSS modules work in this repo, so now instead of having to change anything in webpack.config.js, all you have to do is rename the file to `.module.scss`, and it'll automatically get detected as a CSS module.